content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+also use `-S` (`--set-param`) to set a value for the parameter.
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+$ dvc exp run --queue -S model.conv_units=64
+Queued experiment '30eb9b2' for future execution.
+$ dvc exp run --queue -S model.conv_units=128
+Queued experiment 'ac66940' for future execution.
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '8bb6049' for future execution.
+```
+
+Next, run all (`--run-all`) queued experiments in parallel (using `--jobs`):
+
+```dvc
+$ dvc exp run --run-all --jobs 2
+```
+
+It runs all the experiments, with the specified number of processes in parallel.
+
+## Comparing experiments
+
+The experiments are run several times with different parameters. We use
+`dvc exp show` to compare all of these experiments. This command presents the
+parameters and metrics produced in experiments in a nicely formatted table.
+
+```dvc
+$ dvc exp show
+```
+
+![](/img/start-dvc-exp-show-210704.png)
+
+By default it shows all the parameters and the metrics with the timestamp. If
+you have large number of parameters, metrics or experiments, this may lead to a
+cluttered view. You can limit the table to specific metrics, or parameters, or
+hide the timestamp column with `--include-metrics`, `--include-params`, or
+`--no-timestamp` options of the command, respectively.
+
+```dvc
+$ dvc exp show --no-timestamp \
+  --include-params model.conv_units --include-metrics acc
+```
+
+![](/img/start-dvc-exp-show-no-timestamp-210704.png)
+
+## Persisting experiments
+
+After selecting a experiments from the table, you can commit the hyperparameters
+and other dependencies that produced this successful experiment to your Git
+history.
+
+`dvc exp apply` brings back all specific artifacts and parameters from the
+experiment to the <abbr>workspace</abbr>.
+
+```dvc
+$ dvc exp apply exp-cb13f
+Changes for experiment 'exp-cb13f' have been applied to your current workspace.
+```
+
+We can see the changes in the repository and commit them to Git.
+
+```dvc
+$ git diff
+$ git add .
+$ git commit -m "Successful experiment"
+```
+
+It may also be desirable to commit a particular experiment to a Git branch
+directly, without bringing to the workspace.
+
+```dvc
+$ dvc exp branch exp-05e87 "cnn-256"
+Git branch 'cnn-256' has been created from experiment 'exp-05e87'.
+To switch to the new branch run:
+
+        git checkout cnn-256
+```
+
+You can then checkout and continue from working this branch as usual.
+
+## Note on experiment names
+
+When you create an experiment, DVC generates a hash value from the contents of
+the experiment. This is shown when you use `--queue` option, e.g.,
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+```
+
+After _running_ the experiment, DVC uses another auto-generated name to refer to
+the experiment. Typically these start with `exp-`, and can be set via
```

Comment:
Maybe this whole section can be a `<details>` in the **Comparing experiments** section. The first `exp show` table (img) can have some queued experiments, and after that this hidden part will explain the differences for those interested in clicking on it. Then there's no need for a 3rd table in here.

> Also, it's fine to exclude queued exps from the 2nd table even if the continuity isn't perfect.

Reply:
Removed completely.